eea/facetednavigation/browser/javascript/expand.js

```diff
@@ -73,26 +73,28 @@ jQuery.fn.collapsible = function(settings){
       self.bind(options.events.colapse, function(evt, data){
         options.handle_colapse(evt, data);
       });
-
-      // More/Less button
-      var link = jQuery('<a>').attr('href', '#').text('More');
-      self.button = jQuery('<div>')
-        .addClass('faceted-checkbox-more')
-        .append(link)
-        .hide();
-      self.append(self.button);
-
-      link.click(function(){
-        if(self.colapsed){
-          self.trigger(options.events.expand);
-        }else{
-          self.trigger(options.events.colapse);
+      // Only add button if none exits
+      if (jQuery('.faceted-checkbox-more').length < 1){
```

Comment:
@Gomez :thinking: Hmm, what do you do if you have 2 or more checkbox widgets on the same page?

Reply:
Yeah, its not working and i found another bug :(

Is there a way to reset the eea state with a reset button from outside? Without reload.

Reply:
@Gomez I think you can write directly in Faceted.Query :thinking: and then trigger some events:

https://github.com/eea/eea.facetednavigation/blob/bb85f41f45ea26561c0e306a13975f5b02a0e6d7/eea/facetednavigation/browser/javascript/view.js#L184

See if there is one that can help:

https://github.com/eea/eea.facetednavigation/blob/bb85f41f45ea26561c0e306a13975f5b02a0e6d7/eea/facetednavigation/browser/javascript/view.js#L5-L20

Reply:
Maybe this one:

```javascript
jQuery(Faceted.Events).trigger(Faceted.Events.RESET); 
```

Reply:
Thanks! "QUERY_INITIALIZED" was the missing piece. 

This works:

```
$('.button.reset').on('click', function (e) {
    jQuery(Faceted.Events).trigger(Faceted.Events.RESET);
    Faceted.Query.b_start = 0;
    Faceted.b_start_changed = true;
    Faceted.URLHandler.set();

   jQuery(Faceted.Events).trigger(Faceted.Events.QUERY_INITIALIZED);
}
```